server/dhcp.py

```diff
@@ -224,6 +224,7 @@ class DHCPServerConfiguration(object):
     dhcp_acknowledge_after_seconds = 10
     length_of_transaction = 40
 
+    bind_address = ''
     network = '192.168.173.0'
```

Comment:
Do you know why network has this value?

Reply:
I don't know. I didn't change this and to be honest I don't know where it is used.